Testing-Email-functionality.md

```diff
@@ -0,0 +1,19 @@
+To test the email functionality of Oppia, you can follow these steps:
+
+## Testing with App Engine Deployment
```

Comment:
@Ash-2k3 This is a very heavyweight way to test. I suggest swapping the two sections so that the local version comes first, and not calling the local version a "workaround". In fact I think I might suggest dropping the first section altogether since that is quite an impractical way of testing.

Also could you please verify the local version of the instructions and update as needed? The given file doesn't exist in the codebase any more, and I think the dev_mode_email_services.py should "just work" out of the box. I think it would be helpful to walk through this with a specific example with screenshots, since many devs don't seem to know how to use the local server terminal to read logs...